docs/operator/api.md

```diff
@@ -695,7 +695,7 @@ VMAgentRemoteWriteSettings - defines global settings for all remoteWrite urls.
 | showURL | Whether to show -remoteWrite.url in the exported metrics. It is hidden by default, since it can contain sensitive auth info | *bool | false |
 | tmpDataPath | Path to directory where temporary data for remote write component is stored (default vmagent-remotewrite-data) | *string | false |
 | flushInterval | Interval for flushing the data to remote storage. (default 1s) | *string | false |
-| label | Optional labels in the form &#39;name=value&#39; to add to all the metrics before sending them | map[string]string | false |
+| label | Labels in the form &#39;name=value&#39; to add to all the metrics before sending them. This overrides the label if it already exists | map[string]string | false |
```

Comment:
@Amper @Haleygo should we move this change to k8s operator instead https://github.com/VictoriaMetrics/operator ?

Reply:
Yes, we should

Reply:
@Amper ptal https://github.com/VictoriaMetrics/operator/pull/751